Gemfile

```diff
@@ -2,6 +2,9 @@ source "https://rubygems.org"
 
 gemspec
 
+gem 'websocket-eventmachine-server', require: false
+gem 'websocket-eventmachine-client', require: false
+
```

Comment:
they are test dependencies. no need to be part of gemspec i suppose.

invoker doesn't have to understand websockets. if server keeps tcp connection open, invoker is transparently keeping that connection open. this behavior saved lots of coding.